filebeat/files/filebeat.jinja

```diff
@@ -1,73 +1,27 @@
     -
+      # Paths that should be crawled and fetched. Glob based paths.
+      # For each file found under this path, a harvester is started.
       paths:
-{%- for path in log_path.paths %}
-        - {{ path }}
-{%- endfor %}
-      input_type: {{ log_path.get('input_type', 'log') }}
-      document_type: {{ log_path.get('document_type', 'syslog') }}
-      ignore_older: {{ log_path.get('ignore_older', '24h') }}
-      scan_frequency: {{ log_path.get('scan_frequency', '10s') }}
-      backoff: {{ log_path.get('backoff', '1s') }}
-      max_backoff: {{ log_path.get('max_backoff', '10s') }}
-      backoff_factor: {{ log_path.get('backoff_factor', '2') }}
-      force_close_files: {{ log_path.get('force_close_files', 'false') }}
-      fields_under_root: {{ log_path.get('fields_under_root', 'false') }}
-      close_older: {{ log_path.get('close_older', '1h') }}
-{%- if 'multiline' in log_path %}
-      multiline:
-        pattern: "{{ log_path.multiline.get('pattern', '') }}"
-        negate: {{ log_path.multiline.get('negate', 'false') }}
-        match: {{ log_path.multiline.get('match', 'after') }}
-{%- if 'include_lines' in log_path %}
-      include_lines: ["{{ log_path.include_lines|join('", "') }}"]
-{%- endif %}
-{%- if 'exclude_lines' in log_path %}
-      exclude_lines: ["{{ log_path.exclude_lines|join('", "') }}"]
-{%- endif %}
-{%- endif %}
-{%- if 'fields' in log_path %}
-      fields:
-{%- for field in log_path.fields %}
-{%- for key, value in field.items() %}
-        {{ key }}: {{ value }}
-{%- endfor %}
-{%- endfor %}
-{%- endif %}
-{%- endfor %}
+        - "/var/log/*.log"
+        #- c:\programdata\elasticsearch\logs\*
 
-  registry_file: /var/lib/filebeat/registry
+      # Type of the files. Based on this the way the file is read is decided.
+      # The different types cannot be mixed in one prospector
+      #
+      # Possible options are:
+      # * log: Reads every line of the log file (default)
+      # * stdin: Reads the standard in
+      input_type: log
 
-output:
-{%- set elasticsearch = filebeat.get('elasticsearch', {}) %}
-{%- set logstash = filebeat.get('logstash', {}) %}
-
-{%- if elasticsearch.get('enabled', False) %}
-  elasticsearch:
-    hosts: ["http://{{ elasticsearch.server }}"]
-{%- endif %}
-
-{%- if logstash.get('enabled', False) %}
-  logstash:
-    hosts: ["{{ logstash.server }}"]
-    worker: 1
-    loadbalance: true
-    index: filebeat
-
-{%- if 'tls' in logstash %}
-{%- if logstash.tls.get('enabled', False) %}
-    tls:
-      certificate_authorities: ["{{ logstash.tls.ssl_cert_path }}"]
-{%- endif %}
-{%- endif %}
-{%- endif %}
-
-shipper:
-
-logging:
-  files:
-    rotateeverybytes: 10485760 # = 10MB
+  # Configure what outputs to use when sending the data collected by the beat.
+  # Multiple outputs may be used.
+  output:
+    ### Elasticsearch as output
+    elasticsearch:
+      # Array of hosts to connect to.
+       hosts: ["192.168.1.42:9200"]
```

Comment:
its a pity you lose the benefits of the default values. not sure how to get around this.